dev-requirements.txt

```diff
@@ -22,7 +22,6 @@ pytz
 tox>=3.2.0
 types-requests
 
-dbt-spark==1.6.0rc1
```

Comment:
I can revert if you like.  This was just redundant with requirements.txt.  We can rewrite dev-requirements so that it installs requirements when its installed, btw.